modules/ROOT/pages/authentication-and-authorization/configuration.adoc

```diff
@@ -1,4 +1,5 @@
 = Configuration
+:page-aliases: authentication-and-authorization/index.adoc
```

Comment:
I don't think it's necessary to do this here? If you see the live website, this page already has options to the other versions https://neo4j.com/docs/graphql/3/authentication-and-authorization/configuration/

Reply:
it's going to the deprecated "Auth" page instead of "Authentication and Authorization" (which is just the nav entry, the first actual page is "Configuration")

checked locally, adding the page-alias unfortunately doesn't get rid of the deprecated "Auth" alias (it's not explicitly stated in that file - how does it work then?)

Reply:
Ah yes, you are right about it taking to the deprecated Auth page... but then this was set in the aliases in version 5 already --> https://github.com/neo4j/docs-graphql/edit/5.x/modules/ROOT/pages/authentication-and-authorization/index.adoc

Reply:
I might be missing some information here, but this page exists in all versions (3.x, 4.x. 5.x) so adding this alias has no effect on the version selector.

(This alias would also add a redirect from https://neo4j.com/docs/graphql/3/authentication-and-authorization/ to https://neo4j.com/docs/graphql/3/authentication-and-authorization/configuration/ but we discard the redirect files that Antora adds, and use our own redirects, so this also has no effect.)